Lib/nturl2path.py

```diff
@@ -3,7 +3,12 @@
 This module only exists to provide OS-specific code
 for urllib.requests, thus do not use directly.
 """
-# Testing is done through test_urllib.
+# Testing is done through test_nturl2path.
+
+import warnings
+
+
+warnings._deprecated(__name__, remove=(3, 19))
```

Comment:
Do we have a way to list the alternative in the warning message? Finding the relevant entry in What's New isn't trivial.